src/javascript/app_2/App/Components/Elements/Tabs/tabs.jsx

```diff
@@ -0,0 +1,53 @@
+import classNames  from 'classnames';
+import React       from 'react';
+import PropTypes   from 'prop-types';
+import TabsWrapper from './tabs_wrapper.jsx';
+
+class Tabs extends React.PureComponent {
```

Comment:
@zakibinary 
since it just keeps a state, you can merge its contents with `TabsWrapper` component and remove this file.

Reply:
@zakibinary 
Unfortunately, I made a mistake about this component, would you plz revert your changes. 
Sorry for the inconvenience 😞